be/src/exprs/vectorized/json_functions.cpp

```diff
@@ -221,7 +221,9 @@ ColumnPtr JsonFunctions::get_json_string(FunctionContext* context, const Columns
     auto paths = columns[1];
 
     jsons = json_query(context, Columns{jsons, paths});
-    return json_string(context, Columns{jsons});
+    auto strs = json_string(context, Columns{jsons});
+
+    return _unquote_string(context, Columns{strs});
```

Comment:
what if the JSON string contains a valid quote ?

Reply:
This would only trim the first/last quote of string. So the valid quote in the middle of string would remain.

Reply:
https://github.com/StarRocks/starrocks/blob/6758cc72b5461851ebe79b4dc40a2b7b8226a3e2/be/test/exprs/vectorized/json_functions_test.cpp#L141
A corresponding test case is added here.